contents/docs/feature-flags/common-questions.mdx

```diff
@@ -1,90 +1,178 @@
+If your [release conditions](/docs/feature-flags/creating-feature-flags#release-conditions) depend on user or group properties, and you immediately fetch the feature flag after updating any of these properties, the properties may not be ingested in time to calculate the correct flag value.
 
-2. As per the [advanced configuration](/docs/libraries/js#advanced-configuration), set the `advanced_disable_feature_flags` setting to `true`. This stops automatically requesting feature flags. Instead, use the [bootstrap option](/docs/libraries/js#bootstrapping-flags) to load flags only once, via (server-side evaluation)[/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation], which makes no requests for flags from the client side.
+In this case, when making your `getFeatureFlag()` call, you can manually include in these properties in the method arguments.
 
-### Local evaluation
+### 5. Raise a support ticket
 
-If your bill is high because of too many local evaluation requests, here's how to fix it:
+If none of the above steps resolve your issue, you can [raise a support ticket](https://app.posthog.com/home#supportModal) to see if we can help debug.
 
-When you use local evaluation, you aren't charged for evaluating flags per user (as long as you only evaluate locally), but for requests for flag definitions. Every PostHog instance you initialize creates a poller that sends requests to us every 30 seconds by default.
+import FAQFalseOrNoneEvents from "./snippets/faq-false-or-none-events.mdx"
 
-So, for example, if you have 25 backend servers, and each of those has a `pollInterval` of 30 seconds, the total number of requests per minute you're charged for is 25*2 = 50 requests / minute. In other words, that is 50*60*24*30 =~ 2 million requests / month.
+<FAQFalseOrNoneEvents />
 
-To fix this, you can set the `pollInterval`, or the equivalent parameter in your SDK of choice to 300 seconds instead of 30 seconds. The ramifications here are that whenever you make an update to a feature flag using the PostHog UI, it takes 300 seconds, instead of 30, for that change to roll to all your servers.
+## How can I reduce the latency of feature flag requests on my server?
 
-> **Note:** Do not use local evaluation in an edge / lambda environment, because this initializes a PostHog instance on every call, which can raise your bill drastically. It's best to use regular flag evaluation here.
+Evaluating feature flags requires making a request to PostHog. However, you can [evaluating feature flags locally](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation) instead – without having to make a request to PostHog.
```

Comment:
```suggestion
Evaluating feature flags requires making a request to PostHog. However, you can [evaluate feature flags locally](/docs/feature-flags/bootstrapping-and-local-evaluation#server-side-local-evaluation) instead – without having to make a request to PostHog.
```

Reply:
Maybe also mention client-side flags are already locally evaluated.

Reply:
cached, not locally evaluated, as local evaluation = fresh results computed right now, whereas client side they return immediately, but can be very stale, depending on when you last reloaded them